o1vm/src/interpreters/riscv32im/interpreter.rs

```diff
@@ -2146,7 +2146,17 @@ pub fn interpret_mtype<Env: InterpreterEnv>(env: &mut Env, instr: MInstruction)
 
     match instr {
         MInstruction::Mul => {
-            unimplemented!("Mul")
```

Comment:
it would be nice to comment the implementation from https://msyksphinz-self.github.io/riscv-isadoc/html/rvm.html